deploy/helm/sumologic/values.yaml

```diff
@@ -5014,6 +5078,59 @@ otellogs:
           - type: move
             from: body.log
             to: body
+      journald:
+        directory: /var/log/journal
+        ## This is not a full equivalent of fluent-bit filtering as fluent-bit filters by `_SYSTEMD_UNIT`
+        ## Here is filtering by `UNIT`
+        units:
```

Comment:
Does this list mean that if the user wants to customize this list, they just need to overwrite this property in their `values.yaml` file? 😱  For example, if they want to exclude a specific unit, they need to replicate this whole list with the one unit excluded.
I don't know what the common use cases are (if there are any). Maybe there are no good reasons to customize this list. In that case, do we need to expose it?

Reply:
Yes, this PR is fluent-bit replacement. There is a lot of improvements we can take, but maybe there should be separate issues, so it will be more organised way to track them?

Reply:
> I don't know what the common use cases are (if there are any).

I don't think there are any, but we need to have some plan in case they'll appear